rules/config

```diff
@@ -178,7 +178,7 @@ INCLUDE_ROUTER_ADVERTISER ?= y
 
 # INCLUDE_KUBERNETES - if set to y kubernetes packages are installed to be able to
 # run as worker node in kubernetes cluster.
-INCLUDE_KUBERNETES ?= n
+INCLUDE_KUBERNETES ?= y
```

Comment:
>y [](http://example.com/codeflow?start=22&length=1)

This is behavior change. Other changes in this file is refactoring. Could you separate into multiple PRs?

Reply:
Yes, I just enable it for this moment to check PR build tests pass rate. Should disable it as default.